python/ray/data/read_api.py

```diff
@@ -1212,6 +1240,13 @@ def from_pandas(
 ) -> Dataset[ArrowRow]:
     """Create a dataset from a list of Pandas dataframes.
 
+    .. note::
+        This dataset will be an eagerly computed dataset, where all subsequent
+        transformations will execute eagerly rather than lazily. If you want these
+        transformations to be lazy in order to benefit from better performance and
+        memory stability, convert it to a lazy dataset via
+        :meth:`ds = ds.lazy() <ray.data.Dataset.lazy>` first.
```

Comment:
Could we fix these in this PR? Is it just a matter of adding `.lazy()` to the end of the return expression of this method?

Reply:
@ericl This is being fixed in a follow-up PR, can we not block this broader PR on this?